tests/python/unittest/test_operator.py

```diff
@@ -6957,6 +6957,21 @@ def test_invalid_depth_dim():
     test_invalid_block_size()
     test_invalid_depth_dim()
 
+@with_seed()
+def test_max_pooling_pad_type_same():
+    import math
+    input_data = mx.nd.array(np.random.rand(1,1,10))
+    stride = 2
+    kernel = 2
+    output_data=mx.nd.Pooling(
+        input_data,
```

Comment:
Do you also want to test the case where users selects "same" pad type but also supplies pad values?